coalib/results/TextPosition.py

```diff
@@ -31,3 +31,43 @@ def line(self):
     @property
     def column(self):
         return self._column
+
+    def __le__(self, other):
+        """
+        Performs comparison between two TextPositions, logically equivalent
+        to (self <= other). If the column in a TextPosition is None, consider
+        whole line. If the line in a TextPosition is None, consider whole file.
+
+        :param other: TextPosition to compare with.
+        :return:      Boolean value logically equivalent to (self <= other).
```

Comment:
Also I think you shuld rephrase the `:return:` description to something like 
```
Whether this ``TextPosition`` is behind the other one or the same.
```
or similar :)